apm-sniffer/apm-agent-core/pom.xml

```diff
@@ -53,6 +53,11 @@
     </properties>
 
     <dependencies>
+        <dependency>
+            <groupId>org.apache.skywalking</groupId>
+            <artifactId>apm-toolkit-trace</artifactId>
+            <version>8.2.0-SNAPSHOT</version>
+        </dependency>
```

Comment:
This depedency is not allowed, related to you are still using annotation.

Reply:
If i was a user,i would to add ignore exception by annotation,this would be easy to control rather than change env config frequency. in our department,skywalking is throught as a platform, env is a only way to custom config.But env config is out of user code config, user would not like to frequency config.Env should keep stable.

that is why i want to keep annotation . Could i extract a common light sdk for tookit and agent together?

Reply:
If you want to use annotation, consider using toolkit + activation mode. My point is, the core should not depend on the toolkit. It is not safe in the class loader manipulation.